### ISSUE TITLE
Second run saves output in wrong results directory

### ISSUE THREAD

**Issue by abisi** [MouseLand/Kilosort#603]
### Describe the issue:

I ran kilosort from the GUI, choosing from the dialog the binary file and the results directory.
After a first run on a dataset, the next run on a second dataset will save the results in the output directory of the first run, even if I change manually the results directory everytime. 

I am not sure whether this weird behaviour is the result of a wrong usage of the GUI, or if it's a bug. For example, should I start a new GUI instance everytime, clear the cache and/or reset the GUI after each run?

Thanks in advance for your help!

**Comment by jacobpennington** [comment]
Hello,
This was a bug, you should not need to reset or start a new GUI instance between sorting. I just pushed a fix, please try pulling the newest version. Thanks for catching this!

**Comment by abisi** [comment]
Hello,

I pulled the latest version and installed kilosort again, but the issue still persists.
I opened a new GUI instance and ran kilosort. The saving path shown at the end of the message log box corresponds to the path used in the previous run.

(On a side note, much less critical, the text fields in the Settings window could benefit from being slightly larger, in order to be able to read more of the text: binary file, results directory, sampling frequency. ;) )
